src/lib/traitify-ui.js

```diff
@@ -66,6 +67,7 @@ export default class TraitifyUI {
           return reject("Your target element could either not be selected or was not provided");
         }
 
+        ceaou;        
```

Comment:
This was an error, but was taken out in another pull request that came after this one.